scripts/data/redirects.json

```diff
@@ -11,8 +11,8 @@
   "before": "/photos",
   "after": "http://pics.williamyoumans.com"
 }, {
-  "before": "/blog",
-  "after": "/blog/detail/part-1-interactive-javascript-map-of-canada-with-raphael"
+  //"before": "/blog",
+  //"after": "/blog/detail/part-1-interactive-javascript-map-of-canada-with-raphael"
```

Comment:
JSON doesn't actually support comments. Let's remove this array element entirely. Including brackets.